flutter-idea/README.md

```diff
@@ -0,0 +1,19 @@
+# Gradle Module Structure
+
+This module is a mirror of flutter-intellij (the non-Gradle version) that has sources structured
+in a way that the IntelliJ plugin for Gradle expects. A new module was required because the
+flutter-studio module has dependencies on a number of classes defined in this module and there was a
+circular dependency when those definitions were rooted in flutter-intellij.
+
+The original flutter-intellij structure has been preserved as a courtesy to the engineers that
+already have it checked out. The original metadata files are checked in to the repository and will
+be used when the project is opened. To use the Gradle structure, import
+flutter-intellij/build.gradle instead of opening the project. That will rebuild the metadata
+files according to the Gradle structure.
+
+## Building the plugin
+
+Make sure the proper IDE is unpacked in the artifacts directory. Open a terminal and do
```

Comment:
'Make sure the proper IDE is unpacked in the artifacts directory' - is this a manual step that engineers have to perform, or one handled by a script?